index.js

```diff
@@ -1576,11 +1573,17 @@ class Command extends EventEmitter {
         if (args.length > this._args.length) {
           this._excessArguments(args);
         }
+        return args;
+      };
 
-        this._actionHandler(args);
+      const commandEvent = `command:${this.name()}`;
+      if (this._actionHandler) {
+        checkForUnknownOptions();
+        this._actionHandler(getCheckedArguments());
         if (this.parent) this.parent.emit(commandEvent, operands, unknown); // legacy
       } else if (this.parent && this.parent.listenerCount(commandEvent)) {
         checkForUnknownOptions();
+        getCheckedArguments();
```

Comment:
The name `getXXX` is a function to get something.

`this._actionHandler(getCheckedArguments());`.
For this code, it makes sense.

However, the code here does not use a return value.
I thought it was hard to understand what the 'get' was for.

Reply:
Thanks. I'll take another look. I started without it and added it to make the "get" usage make more sense. I'll try separating out the check and the variadic support since that is only needed in one case.

Reply:
Reworked. Longer but hopefully clearer.

Reply:
Thanks.